deployments/pulse3d/services/pulse3d/src/requirements.txt

```diff
@@ -10,7 +10,6 @@ starlette==0.16.0
 typing-extensions==4.0.1
 uvicorn[standard]==0.16.0
 stream-zip==0.0.48
-pulse3d==0.27.0
```

Comment:
Should this be updated to a new version instead of deletion?

Reply:
@NikitaPoly The reason we want to store peaks and valleys in parquet files is to remove this dependency from the pulse3d service so that we don't have to worry about the various versions. So this PR removes anything being used from Pulse3D and is no longer being imported for anything. It was initially only being used for `peak_detector` in interactive analysis

Reply:
I think it makes sense to keep it as an import so we can keep a single source of truth for the constants that we import from it. But then again, these could change in new pulse3d versions, so it might actually make more sense to not have them in this svc at all and grab them in the job instead